google/cloud/security/scanner/audit/org_rules_engine.py

```diff
@@ -459,10 +480,8 @@ def _dispatch_rule_mode_check(self, mode, rule_members=None,
             rule_members=rule_members,
             policy_members=policy_members)
 
-    # TODO: Investigate making a function, not a method.
-    # pylint: disable=no-self-use
-    def _whitelist_member_check(self, rule_members=None,
-                                policy_members=None):
+    @staticmethod
```

Comment:
I'm not sure the best solution is protected static methods, it looks like trying to use a class as a namespace, which is not really the intent.

Since they can be called and tested independent of their class, I would move these out of the class(es) they are defined in, to top level protected functions. If you want them in a different namespace for readability, then I'd move them into their own class that only contains static methods, or a new .py module, but a better solution is to make sure they are named appropriately (which I think they already are).